src/components/Selector/Selector.js

```diff
@@ -181,7 +181,7 @@ const Selector = ({
   const hasValue = !isEmpty(selectedOption);
   const hasOptions = !!options?.length;
   const disabled = !hasOptions && !allowEnteringCustomAddress;
-  const placeholderText = !disabled ? `${placeholder}...` : 'no options to select';
+  const placeholderText = !disabled ? `${placeholder}...` : t('label.noOptionsToSelect');
```

Comment:
shouldn't the dots be somewhere in translation?

Reply:
Oh, yeah. I'll add them as suffix. Thanks! :)